sirepo/template/elegant.py

```diff
@@ -1147,6 +1148,7 @@ def _fix(v):
                         field_range[col][1] = max(_fix(max(values)), field_range[col][1])
                     else:
                         field_range[col] = [_fix(min(values)), _fix(max(values))]
+            lattice_id = LatticeUtil.get_lattice_id_from_file_id(data, file_id)
```

Comment:
No need for the local variable lattice_id. Just call the function when building the PKDict